src/Compare.js

```diff
@@ -1,25 +1,28 @@
 import React, { Component } from 'react';
-import characters from './data';
+import { characters, categories } from './data';
 import CompareItem from './CompareItem'
 import './styles/App.css';
 
+
 class Compare extends Component {
   constructor() {
     super();
     this.state = {
-      data: characters.slice(0, characters.length)
+      data: characters.slice(0, characters.length),
+      highlight: 'Name'
```

Comment:
Ah, interesting adding the highlight in the constructor. No edit, just wanted to comment!